deploy/downstream-prep.sh

```diff
@@ -1,99 +1,184 @@
 #!/bin/bash
 
-#Declare v1.0 image information
-V1_0_IMAGES=(
-  "controller"
-  "operator"
-  "plugin"
-  "ui"
-  "velero"
-  "helper"
-)
-
-declare -A V1_0_IMG_MAP
-V1_0_IMG_MAP[controller_repo]="openshift-migration-controller"
-V1_0_IMG_MAP[operator_repo]="openshift-migration-operator"
-V1_0_IMG_MAP[plugin_repo]="openshift-migration-plugin"
-V1_0_IMG_MAP[ui_repo]="openshift-migration-ui"
-V1_0_IMG_MAP[velero_repo]="openshift-migration-velero"
-V1_0_IMG_MAP[helper_repo]="openshift-migration-velero-restic-restore-helper"
-
-#Remove latest and stable as options
+#Remove latest as option
 if grep -q latest deploy/olm-catalog/mig-operator/mig-operator.package.yaml; then
   sed -i 3,4d deploy/olm-catalog/mig-operator/mig-operator.package.yaml
 fi
 sed -i s,mig-operator,cam-operator,g deploy/olm-catalog/mig-operator/mig-operator.package.yaml
 rm -rf deploy/olm-catalog/mig-operator/stable deploy/olm-catalog/mig-operator/latest
 
-#Get latest 1.0 images
-for i in ${V1_0_IMAGES[@]}; do
-  docker pull registry-proxy.engineering.redhat.com/rh-osbs/rhcam-${V1_0_IMG_MAP[${i}_repo]}:v1.0 > /dev/null
-done
+if [ -d deploy/olm-catalog/mig-operator/v1.1.0 ]; then
+  #Declare v1.1 image information
+  V1_1_IMAGES=(
+    "controller"
+    "operator"
+    "plugin"
+    "ui"
+    "velero"
+    "helper"
+    "gcpplugin"
+    "awsplugin"
+    "azureplugin"
+  )
+
+ declare -A V1_1_IMG_MAP
+  V1_1_IMG_MAP[controller_repo]="openshift-migration-controller"
+  V1_1_IMG_MAP[operator_repo]="openshift-migration-operator"
+  V1_1_IMG_MAP[plugin_repo]="openshift-migration-plugin"
+  V1_1_IMG_MAP[ui_repo]="openshift-migration-ui"
+  V1_1_IMG_MAP[velero_repo]="openshift-migration-velero"
+  V1_1_IMG_MAP[helper_repo]="openshift-migration-velero-restic-restore-helper"
+  V1_1_IMG_MAP[gcpplugin_repo]="openshift-migration-velero-plugin-for-gcp"
+  V1_1_IMG_MAP[awsplugin_repo]="openshift-migration-velero-plugin-for-aws"
+  V1_1_IMG_MAP[azuerplugin_repo]="openshift-migration-velero-plugin-for-microsoft-azure"
```

Comment:
These names need to match watch @dymurray requested for downstream repo names here and elsewhere.

Reply:
This is what I used. Just waiting on the ticket to be worked.